client/lib/billing-card/app/controllers/overlays/billing-card.js

```diff
@@ -0,0 +1,131 @@
+import Ember from 'ember';
+import TaskController from 'tahi/pods/task/controller';
+
+export default TaskController.extend({
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected '(end)' and instead saw 'default'.